Source/WebCore/Modules/credentialmanagement/CredentialsContainer.h

```diff
@@ -28,8 +28,10 @@
 
 #if ENABLE(WEB_AUTHN)
 
+#include "ActiveDOMObject.h"
 #include "AuthenticatorCoordinator.h"
 #include "DigitalCredential.h"
+#include "Document.h"
```

Comment:
Anne also called this out separately, but I'd guess both of these additions shouldn't needed.

Reply:
yep 🙈